python/test/compose_test.py

```diff
@@ -0,0 +1,99 @@
+""" cloudstorage compose tester"""
+import webapp2
+import logging
+import re
+import cloudstorage
+from google.appengine.api import app_identity
```

Comment:
Lets get Johns input on that as well but I fear that you many need to maintain the package hackery seen in other tests.